### ISSUE TITLE
deprecated in favor of django.utils.functional.keep_lazy()

### ISSUE THREAD

**Issue by dimitrismakris** [django-crispy-forms/django-crispy-forms#727]
* Package version: 1.6.1
* Django version: 1.11.2
* Python version: 3.5.2

### Description:

I tried to upgrade my Django version from 1.8 to 1.11.2 but now I get the following error:

```
/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/crispy_forms/templatetags/crispy_forms_utils.py:19: RemovedInDjango20Warning: django.utils.functional.allow_lazy() is deprecated in favor of django.utils.functional.keep_lazy()
  remove_spaces = allow_lazy(remove_spaces, text_type)
```

**Comment by carltongibson** [comment]
That's just a warning. 2.0 is not due until December. We have a new release in the pipeline before then.

**Comment by japrogramer** [comment]
@carltongibson cool, I started testing django 2.0 and aside from the import error for urlresolver, Im also getting an import error for allow_lazy. when Is the new release available? maybe a compat.py file should be included 
here is an example
https://github.com/pytest-dev/pytest-django/blob/master/pytest_django_test/compat.py

a quick find shows that this approach is popular

```
$find . -type f -name 'compat.py'

./allauth/compat.py
./billiard/compat.py
./billiard/tests/compat.py
./celery/utils/compat.py
./chardet/compat.py
./debug_toolbar/compat.py
./django_extensions/compat.py
./factory/compat.py
./faker/compat.py
./graphene/pyutils/compat.py
./graphene_django/compat.py
./idna/compat.py
./kombu/compat.py
./kombu/utils/compat.py
./promise/compat.py
./pytest_django/compat.py
./requests/compat.py
./sphinx/util/compat.py
./sqlparse/compat.py
./test_plus/compat.py
./twisted/python/compat.py

```

**Comment by carltongibson** [comment]
This was fixed in bea8490ba5e6128890e9dc0582215fcdb3b83ea2
